source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -201,6 +201,8 @@ export class CustomPropertySheetEntry extends PropertySheetEntry {
 
 export interface IPropertySheetEditorCommand {
     caption: string;
+    altText: string;
+    expanded?: string;
```

Comment:
should this be a `boolean`?

Reply:
(and below)

Reply:
I made it a string because setAttribute takes a string. But you're probably right that this should be a bool and i'll toString it as needed.